applications/system-service/wlan.h

```diff
@@ -33,12 +33,24 @@ class Wlan : public QObject, public SysObject {
         }
         return "";
     }
+    bool pingIP(std::string ip, const char* port) {
+        return !system(("echo -n > /dev/tcp/" + ip.substr(0, ip.length() - 1) + "/" + port).c_str());
+    }
     bool isConnected(){
-        auto ip = exec("ip r | grep " + iface() + " | grep default | awk '{print $3}'");
-        return ip != "" && !system(("echo -n > /dev/tcp/" + ip.substr(0, ip.length() - 1) + "/53").c_str());
+        auto ip = exec("ip r | /bin/grep " + iface() + " | /bin/grep default | awk '{print $3}'");
```

Comment:
What's the reason for using /bin/grep instead of the grep on the path.

Reply:
Because of that: https://github.com/ddvk/remarkable2-framebuffer/issues/37

Reply:
I'd rather they fixed it. Since it's just a warning and grep isn't using the framebuffer it's just causing annoying log output.